styleguide/templates/overview.html.twig

```diff
@@ -0,0 +1,88 @@
+<div class="AknStyleGuide-element" data-group="Overview">
+    <h1 class="AknStyleGuide-elementTitle">
+        Overview
+    </h1>
+
+    <p>
+        In these pages you will find the complete list of available modules for Akeneo PIM. Akeneo PIM uses the
```

Comment:
In the following pages you will find a comprehensive list of available modules for Akeneo PIM. We organize our styles using the BEM methodology